nmdc_server/crud.py

```diff
@@ -485,6 +486,14 @@ def create_bulk_download(
         raise
 
 
+def replace_nersc_data_host(url: str):
```

Comment:
I suggest adding a docstring. Here's an example:

```suggestion
def replace_nersc_data_host(url: str) -> str:
    """
    Updates NERSC URLs so they have the custom prefix defined in
    an environment variable. This can be used to optimize the URLs
    for HTTP clients that have direct access to the NERSC network.
    """
```